XamlControlsGallery/ItemsPageBase.cs

```diff
@@ -71,7 +71,7 @@ protected void OnItemGridViewKeyDown(object sender, KeyRoutedEventArgs e)
             if (e.Key == VirtualKey.Up)
             {
                 var nextElement = FocusManager.FindNextElement(FocusNavigationDirection.Up);
-                if (nextElement.GetType() == typeof(NavigationViewItem))
+                if (nextElement != null && nextElement.GetType() == typeof(Microsoft.UI.Xaml.Controls.NavigationViewItem))
```

Comment:
Nitpick: could just use "nextElement?.GetType()" if you like the shorter syntax.

Reply:
Thank you, I didn't know about this syntax.